recipes/seqwish/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+export LD_LIBRARY_PATH=${PREFIX}/lib
+export LIBRARY_PATH=${PREFIX}/lib
+export LD_LIBRARY_PATH=${PREFIX}/lib
```

Comment:
You’re setting `LD_LIBRARY_PATH` twice (also, is it necessary to set this at all?)

Reply:
Possibly not, will give it a try